.github/workflows/build.yml

```diff
@@ -29,6 +32,10 @@ jobs:
         conda install -n test -q -y c-compiler cython gsl numpy pip pytest setuptools
         python -m pip install -r requirements.txt
```

Comment:
I'm not sure, but I think you might need to still add a `conda activate test` here and as the first stage in each job `run` setting.

Reply:
Or maybe not. Things seem to be passing now!